packages/rocketchat-lib/server/methods/joinRoom.js

```diff
@@ -18,7 +18,7 @@ Meteor.methods({
 		}
 
 		if ((room.joinCodeRequired === true) && (code !== room.joinCode) && !RocketChat.authz.hasPermission(Meteor.userId(), 'join-without-join-code')) {
-			throw new Meteor.Error('error-code-invalid', 'Invalid Code', { method: 'joinRoom' });
+			throw new Meteor.Error('error-code-invalid', 'Invalid Password', { method: 'joinRoom' });
```

Comment:
Shouldn't we say room password or room join code?  I think password here is still just as ambiguous

Reply:
Ok, I'll change to Room Password